src/main/java/com/google/api/gax/grpc/CallContext.java

```diff
@@ -35,31 +35,26 @@
 import io.grpc.Channel;
 
 /**
- * CallContext encapsulates arguments used to make an RPC call.
+ * CallContext encapsulates context data used to make an RPC call.
  *
- * <p>CallContext is immutable in the sense that none of its methods modifies
- * the CallContext itself or the underlying data.
- * Methods of the form {@code withX}, such as {@link #withChannel},
- * return copies of the object, but with one field changed.
- * The immutability and thread safety of the arguments solely depends on the arguments themselves.
+ * <p>CallContext is immutable in the sense that none of its methods modifies the CallContext itself
+ * or the underlying data. Methods of the form {@code withX}, such as {@link #withChannel}, return
+ * copies of the object, but with one field changed. The immutability and thread safety of the
+ * arguments solely depends on the arguments themselves.
  */
-public final class CallContext<RequestT> {
+public final class CallContext {
   private final Channel channel;
   private final CallOptions callOptions;
-  private final RequestT request;
 
-  private CallContext(Channel channel, CallOptions callOptions, RequestT request) {
+  private CallContext(Channel channel, CallOptions callOptions) {
     this.channel = channel;
     this.callOptions = callOptions;
-    this.request = request;
   }
 
-  public static <T> CallContext<T> of(Channel channel, CallOptions callOptions, T request) {
-    return new CallContext<T>(channel, callOptions, request);
-  }
+  public static final CallContext DEFAULT = new CallContext(null, CallOptions.DEFAULT);
```

Comment:
The Java peoples recommended that we make a static accessor function even in cases like this.